packages/perseus/src/types.ts

```diff
@@ -469,15 +476,14 @@ export type VideoKind = "YOUTUBE_ID" | "READABLE_ID";
 // could be used. Aim to shrink the footprint of PerseusDependencies and try to
 // use alternative methods where possible.
 export type PerseusDependencies = {
-    // JIPT
     JIPT: JIPT;
-    graphieMovablesJiptLabels: JiptLabelStore;
-    svgImageJiptLabels: JiptLabelStore;
-    rendererTranslationComponents: JiptTranslationComponents;
```

Comment:
These only make sense when `useJIPT: true`, so I've moved these into the `JIPT` type union when `useJIPT: true`